openquake/commonlib/logictree.py

```diff
@@ -792,6 +796,21 @@ def validate_uncertainty_value(self, node, branchset, value):
                 node, self.filename,
                 'expected a pair of floats separated by space'
             )
+        elif branchset.uncertainty_type == 'incrementalMFDAbsolute':
+            mbr = value.split(',')
+            if len(mbr) == 3:
+                min_mag, bin_width, rates = mbr
+                try:
+                    rates = positivefloat(rates)
```

Comment:
positivefloats is still not imported, so this cannot work. You are getting a `NameError: name 'positivefloats' is not defined` but since you are using a catch all `except` you are not recognizing that.